src/Microsoft.VisualBasic/tests/VB/Microsoft.VisualBasic.VB.Tests.vbproj

```diff
@@ -1,9 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{da95454e-5f0e-4f8a-871a-25d1ab62867a}</ProjectGuid>
-    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <Configurations>netstandard-Unix-Debug;netstandard-Unix-Release;netstandard-Windows_NT-Debug;netstandard-Windows_NT-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="FileIOTestBase.vb" />
+    <Compile Include="Helpers.vb" />
+    <Compile Include="SpecialDirectoriesTests.vb" />
     <Compile Include="Dummy.vb" />
+    <Compile Include="FileIOTests.vb" />
+    <Compile Include="TextFieldParserTests.vb" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.VisualBasic.vbproj" />
```

Comment:
I don't believe there should be an explicit project reference, it's implicitly referencing the implementation. Having said that though this project is in a non standard location.

Reply:
It doesn't build if you can't include the explicit reference, I reported this when I started also the non-standard location causes some issues also reported. I was unable to move it without causing new problems that I filed. The C# and test VB files should probably be in the same directory with 2 project files but this goes against being able to build based on the directory structure without listing the files.